### PR TITLE
I added validation tests for a "programs" start date and end date

### PR DESCRIPTION
**Pull request by erbes**

### CODE REVIEW

**Comment by Ajedi32** [comment]
These changes are in the wrong file... 'edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator' doesn't sound right to me. Shouldn't it be '/src/edu/uwm/cs361/fantastic_five/training_tracker/services/ProgramValidator.java'?

File: edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	public Map<String, List<String>> validate(String name, String instructor, String price, String startDate, String endDate) {
+		errors = new HashMap<String, List<String>>();
+
+		validateName(name);
+		validateInstructor(instructor);
+		validatePrice(price);
+		validateStartDate(startDate);
+		validateEndDate(endDate);
+		return errors;
+	}
```
Comment (Ajedi32) [inline edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator:20]: Changing this method's signature without any corresponding changes to the code that calls this method is going to break lots of stuff.

File: edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		List<String> startDateErrors = new ArrayList<String>();
+
+		if (startDate == null || startDate.isEmpty()) {
+			startDateErrors.add("Start Date must not be blank.");
+		}
+
+		if (!startDateErrors.isEmpty()) errors.put("startDate", startDateErrors);
+		
+	}
+}
```
Comment (Ajedi32) [inline edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator:88]: This looks fine to me. These changes should be accompanied by corresponding changes to the tests in '/src/edu/uwm/cs361/fantastic_five/training_tracker/tests/services/ProgramValidatorTest.java' though.

**Comment by Ajedi32** [comment]
The new lines you wrote here look good, but you made the changes in the wrong file so I can't just merge this straight out. Plus, a few things that need to accompany these changes that you left out.

File: edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator (hunk 2 above)
Comment (erbes) [inline edu/uwm/cs361/fantastic_five/training_trackerapp/entities/ProgramValidator:88]: I have corresponding changes for the porgramValidatorTest

**Comment by Ajedi32** [comment]
Closing as duplicate of #41

**Pull request closed without merging**